doc/source/serve/production.md

```diff
@@ -583,3 +565,217 @@ This section summarizes the best practices when deploying to production:
 * Use `serve config` to check the latest config that your Serve application received. This is its goal state.
 * Make lightweight configuration updates (e.g. `num_replicas` or `user_config` changes) by modifying your Serve config file and redeploying it with `serve deploy`.
 * Make heavyweight code updates (e.g. `runtime_env` changes) by starting a new Ray cluster, updating your Serve config file, and deploying the file with `serve deploy` to the new cluster. Once the new deployment is finished, switch your traffic to the new cluster.
+
+(serve-in-production-kubernetes)=
+
+## Deploying to Production on Kubernetes
+
+The above sections walk through how to deploy a Serve application manually to a running Ray cluster using the Serve CLI.
+Ray Serve can also be run on Kubernetes, providing the best of both worlds: the user experience and scalable compute of Ray Serve and the production-readiness of Kubernetes.
+This also allows you to integrate with existing applications that may be running on Kubernetes.
+The best practice when running on Kubernetes is to use the [RayService] custom resource (CR) that's provided as part of [KubeRay] because it automatically handles important production requirements such as health checking, status reporting, failure recovery, and upgrades.
```

Comment:
Why are there names in brackets throughout this description? E.g. `[RayService]` or `[Kuberay]`?

Reply:
these are links

Reply:
the link for them is defined at the bottom of the page then you can reuse them

Reply:
Oh, that's cool, I didn't know that was a feature.